manifests/charts/gateway/templates/service.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
```

Comment:
Can you add an option to not create the service ? For example if service.type == 'NONE'

Reply:
ack, will add it soon